RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/tmpDeviceCollection.h

```diff
@@ -0,0 +1,16 @@
+#ifndef RecoParticleFlow_PFRecHitProducerAlpaka_interface_alpaka_tmpDevice_h
+#define RecoParticleFlow_PFRecHitProducerAlpaka_interface_alpaka_tmpDevice_h
+
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/tmpSoA.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::reco {
+
+  using tmpDeviceCollection = PortableCollection<::reco::tmpSoA>;
```

Comment:
These should either use more descriptive name, or more specific namespace (and start with an uppercase letter).

Reply:
These `tmp` collections were renamed to `ClusteringVars` and `ClusteringEdgeVars`, is this sufficient?